docs/articles/documentation/test-api/assertions/index.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: docs
+title: Assertions
+permalink: /documentation/test-api/assertions/
+checked: false
+---
+# Assertions
+
+To check if the state of the tested webpage matches the one you expect to see, use *assertions*.
+
+TestCafe provides a comprehensive set of assertions and all of them based on Behavior Driven Development style (BDD-style).
+See [Assertion API](assertion-api.md).
+
+This topic contains the following sections.
+
+* [Using Assertions](#using-assertions)
+* [Assertion timeout](#assertion-timeout)
+* [Assertion options](#assertion-options)
+  * [options.timeout](#optionstimeout)
+
+## Using Assertions
+
+The following example demonstrates how to check whether two values are equal or not.
+
+```text
+await t.expect( actual ).eql( expected, message, options );
+```
+
+Parameter              | Type     | Description
+---------------------- | -------- | -------------------------------------------------------------------------------
+`actual`                 | Any type | The comparison value
+`expected`                 | Any type | Expected value
+`message`&#160;*(optional)* | String   | Assertion message; indicated in the report if there is an error
+`options`&#160;*(optional)* | Object   | See [Options](#assertion-options)
+
+Use assertions to check states of webpage elements and any values of standard types (String, Number, Object and etc.).
+To check the state of any element on the page in an assertion, you can use properties and methods of
+the [DOM Node Snapshot](../selecting-page-elements/selectors.md#dom-node-snapshots)
+or use [Snapshot API Shorthands](../selecting-page-elements/selectors.md#snapshot-api-shorthands).
+
+The sample below demonstrates how to use assertions to check element states.
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `Example page`
+    .page `http://devexpress.github.io/testcafe/example/`;
+
+test('Check property of element', async t => {
+    const developerNameInput = Selector('#developer-name');
+
+    await t
+        .expect(developerNameInput.value).eql('', 'input is empty')
+        .typeText(developerNameInput, 'Peter')
+        .expect(developerNameInput.value).eql('Peter', 'input has text');
+});
+```
+
+## Assertion Timeout
```

Comment:
It's makes `options.timeout` ambiguous. Also it shouldn't be presented as `Assertion Timeout`. Our assertions mechanism should be described in details above.